### PR TITLE
Doc: Add Booster Rockets list

### PR DESCRIPTION
**Pull request by Moneyba**
## Description
There is currently no reference to the Booster Rockets in the documentation.

## Changes
Add a list with the current BoosterRockets

## Checks
- [ ] Project Builds
- [ ] Project passes tests and checks
- [ ] Updated documentation accordingly

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2864,6 +2865,17 @@ Booster.configure('development', (config: BoosterConfig): void => {
...
+#### Booster Rockets list
+
+Here you can find the official Booster Rockets developed at this time:
+
+- [Authentication Booster Rocket for AWS](https://github.com/boostercloud/rocket-auth-aws-infrastructure)
+- [Backup Booster Rocket for AWS](https://github.com/boostercloud/rocket-backup-aws-infrastructure)
+- [Batch File Booster Rocket for AWS](https://github.com/boostercloud/rocket-batch-file-process-aws-infrastructure)
+- [File Uploader Booster Rocket for AWS](https://github.com/boostercloud/rocket-fileuploader-aws-infrastructure)
+- [Static Sites Booster Rocket for AWS](https://github.com/boostercloud/rocket-static-sites-aws-infrastructure)
+- [[WIP\] Kafka Integration Rocket](https://github.com/boostercloud/rocket-kafka-aws-infrastructure)
```
Comment (javiertoledo) [inline docs/README.md:2877]: I deleted a couple of them that are not public yet :trollface: 

```suggestion
- [Authentication Booster Rocket for AWS](https://github.com/boostercloud/rocket-auth-aws-infrastructure)
- [Backup Booster Rocket for AWS](https://github.com/boostercloud/rocket-backup-aws-infrastructure)
- [Static Sites Booster Rocket for AWS](https://github.com/boostercloud/rocket-static-sites-aws-infrastructure)
```

**Review by javiertoledo — CHANGES_REQUESTED** [review]
Just deleted a few from the list that are not public yet. Maybe we should consider publishing the file uploader one 🤔

**Review by AdrianLorenzoDev — APPROVED** [review]
LGTM!

**Comment by Moneyba** [comment]
@javiertoledo, I made that change but I still see "changes requested" and I can't merge. Could you check if it's ok now?

**Review by rdoria1 — APPROVED** [review]
LGTM

**Pull request merged**